src/tracks/playabletrack/wavetrack/ui/WaveTrackControls.cpp

```diff
@@ -810,6 +810,15 @@ void WaveTrackMenuTable::OnMergeStereo(wxCommandEvent &)
    auto partner = static_cast< WaveTrack * >
       ( *tracks.Find( pTrack ).advance( 1 ) );
 
+   if (pTrack->GetRate() != partner->GetRate()) {
+      using namespace BasicUI;
+      ShowMessageBox(XO("Channels must have the same sample rate"),
```

Comment:
This sentence will probably let the user think that she now has to convert one of the tracks to the sample rate of the other. As @crsib pointed out, an alternative worth considering would be panning - simpler, no memory usage increase, lossless.
- [] Please consider suggesting this in your sentence.

Reply:
There is also the danger that the user changes the sample rate of one of the tracks by using the rate sub menu on the track's menu, rather than Tracks > Resample.